playbooks/provisioner/openstack/main.yml

```diff
@@ -4,3 +4,5 @@
   roles:
       - { role: openstack/neutron, action: setup }
       - { role: openstack/nova, action: setup }
+
+- include: plugins.yml
```

Comment:
@tkammer what will define "provisionr.plugins" ? 
Should there be a line in topology/bmc.yaml of "plugins: enabled"